dashboard/backend/handler/apihandler.go

```diff
@@ -0,0 +1,170 @@
+package handler
+
+import (
+	"fmt"
+	"net/http"
+
+	restful "github.com/emicklei/go-restful"
```

Comment:
Any particular reason you picked this package?

Reply:
When I started the UI, my initial thought was that in the end it might end up integrated with the kubernetes dashboard once they have a better plan to support third parties integration, so the backend architecture is pretty close to https://github.com/kubernetes/dashboard/tree/master/src/app/backend. `go-restful` is what is used there.  
Since it seems unlikely that we will go this route anyway, I can use something else if you like.

Reply:
SGTM thanks.

Reply:
Resolved.